drivers/misc/stm32_tsc/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Arif Balik <arifbalik@outlook.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config STM32_TSC
+	bool "STM32 TSC driver"
+	default y
+	depends on DT_HAS_ST_STM32_TSC_ENABLED
+	help
+	  Enable driver for STM32 TSC peripheral.
+
+config STM32_TSC_INTERRUPT
+	bool "Use interrupt for STM32 TSC"
+	default y
+	depends on STM32_TSC
+	help
+	  Use interrupt for STM32 TSC peripheral.
```

Comment:
Why would one set this to 'n'? May be worth a comment (or just drop the option and use it all the time)